MPC/modules/IDLBase.pm

```diff
@@ -363,13 +363,13 @@ sub parse {
     elsif ($str =~ s/^L'(.|\\.|\\[0-7]{1,3}|\\x[a-f\d]{1,2}|\\u[a-f\d]{1,4})'//i) {
       ## Wchar literal
     }
-    elsif ($str =~ s/^@([a-z_]+)(?:\s*\((.*)\))?//) {
-      # Topic Type Annotations
+    elsif ($str =~ s/^@([A-Za-z0-9:_]+)(?:\s*\(([^\)]*)\))?//) {
+      ## Annotation
       my $name = $1;
       if ($name eq 'default_nested' || $name eq 'nested') {
         $cnested = (defined $2 && $2 eq 'FALSE') ? 0 : 1;
       } elsif ($name eq 'topic') {
-        # @topic can have parameters, but we can ignore them here
+        # TODO: Take platform member into consideration
```

Comment:
Is 'platform' in scope?

Reply:
No, but since I was already touching the code to fix a serious problem with the annotation regex I wanted to note that it should probably be supported since `opendds_idl` supports it now.

Reply:
I guess what you're asking is if can be deferred and the answer to that  is yes.